### ISSUE TITLE
Rails 5 support

### ISSUE THREAD

**Issue by malachaifrazier** [mbulat/plutus#72]
I've been struggling to do this myself, but to no avail.

**Comment by mbulat** [comment]
@malachaifrazier This is on my todo list... if you have any errors or issue running it with Rails 5, please post them. It'll help me speed up the process. Thanks!

**Comment by malachaifrazier** [comment]
Will do!

**Comment by malachaifrazier** [comment]
@mbulat I've gotten things banged out here: https://github.com/thiesa/plutus/tree/rails-5-support-2

Unfortunately, `models/tenancy_spec.rb:40` seems to be nondeterministic and failing when I run either the full suite or the `model` directory.

**Comment by mbulat** [comment]
I've pushed a new build with Rails5 support to https://github.com/mbulat/plutus/tree/rails5

Anyone looking for Rails 5 support, please pull that branch and test it... I'll merge into master if no one has issues.

**Comment by inopinatus** [comment]
@mbulat Doesn't work with 5.1 - alias_method_chain was deprecated in 5.0 and is gone in 5.1, in favour of prepend.

**Comment by inopinatus** [comment]
@mbulat Doesn't work with Rails 5.1. alias_method_chain was deprecated in 5.0 and is gone in 5.1, in favour of prepend.

**Comment by mbulat** [comment]
Thanks @inopinatus. That was actually being called in a method we deprecated a couple versions ago, so I've simply removed that method. 

See 0270e1e21b55de7816488870a22c5cd8431bb482

**Comment by abhishiv** [comment]
Hey @mbulat 

Just tied that branch on a fresh install, and I got the following while running the migrations

```
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreatePlutusTables < ActiveRecord::Migration[4.2]
/Users/abhishiv/work/erp/core/db/migrate/20170627015143_create_plutus_tables.rb:1:in <top (required)>
StandardError: Directly inheriting from ActiveRecord::Migration is not supported. Please specify the Rails release the migration was written for:

  class CreatePlutusTables < ActiveRecord::Migration[4.2]
/Users/abhishiv/work/erp/core/db/migrate/20170627015143_create_plutus_tables.rb:1:in <top (required)>
Tasks: TOP => db:migrate
(See full trace by running task with --trace)
```

Using the proper class seems to fix it, so probably it's an easy fix

```
class CreatePlutusTables < ActiveRecord::Migration[4.2]
```

Otherwise it seems to work great! Thanks!

**Comment by abhishiv** [comment]
Ah, just noticed an issue with jquery-ui. Seems #61 is in master, but not in rails5 branch. Updating rails5 branch should fix it, I think.

**Comment by mbulat** [comment]
@abhishiv  I'm not sure what the jquery issue is. I'm not seeing it on my end, and #61 was merged into master. 

I've gone ahead and merged the rails branch into master now... can you update your master branch and see if the issue persists. Thanks!

**Comment by mbulat** [comment]
I'm going to close this for now as rails5 is merged into master now, and released as 0.13. Please re-open if there's a specific Rails 5 problem. Otherwise go ahead and open issue specific threads. Thanks!
